src/Microsoft.Management.UI.Internal/ManagementList/FilterProviders/SearchTextParser.cs

```diff
@@ -121,7 +121,7 @@ protected virtual string GetPattern()
                 patterns.Add(rule.Pattern);
             }
 
-            patterns.Add(string.Format(CultureInfo.InvariantCulture, "(?<{0}>){1}", FullTextRuleGroupName, ValuePattern));
+            patterns.Add(string.Format(CultureInfo.InvariantCulture, $"(?<{FullTextRuleGroupName}>){ValuePattern}"));
```

Comment:
The same. I stop review. Please replace all Format with Create.

Reply:
I'm still working on it, its [WIP]. I'll change it to draft